azure-batch-module/main.tf

```diff
@@ -0,0 +1,71 @@
+/*
+
+# Purpose
+
+Create an Azure Batch account [1] with a pool populated of Docker-compatible compute nodes [2].
+
+# Notes
+
+1. Create a storage account [3] to store files used by the compute nodes to perform any backend 
+   calculation.
+2. Create a container within the above-mentioned storage account with a user-tunable retention 
+   policy of n days [4].
+3. Create the Batch account. Warning: it is not possible to create more then a single Batch Account
+   per subscription and per region due to quota limitations (see Ref. [5]). Consider increasing that
+   if needed.
+4. Create a pool made of Docker-compatible nodes whose size is user-chosen [6]. Warning: by default, 
+   only a very limited array of VMs is available (10 VMs chosen amonst the Av2 Series, DSv3 Series, 
+   Dv3 Series, ESv3 Series and Ev3 Series). Should you need other types of VMs or more of them, 
+   consider asking for an increase of quota for your Azure subscription (see Ref. [5]).
+
+# Usage
+
+module "azure_batch_service" {
+  source              = "./modules/batch-account"
+
+  resource_group_name = azurerm_resource_group.BatchResourceGroup.name
```

Comment:
Since you depend on a resource group here, I would add the creation of it to the example section just before you call this module.

Reply:
Done.